chia/util/chia_logging.py

```diff
@@ -39,6 +40,14 @@ def initialize_logging(service_name: str, logging_config: Dict, root_path: Path)
         )
         logger.addHandler(handler)
 
+    if logging_config["log_syslog"]:
```

Comment:
Can you test this on existing installations that upgrade? we don't modify config.yaml on upgrade so this will probably throw a missing key

Reply:
Thanks BTW!

Reply:
You're right! With and old config it didn't work. I assumed there was a migration taking place because of the _chia init_ message "/Users/gldecurtins/.chia/mainnet already exists, no migration action taken".
I changed it to the .get() method instead and default to False.